### ISSUE TITLE
Adding two SizedArray results in an SArray

### ISSUE THREAD

**Issue by pjentsch0** [JuliaArrays/StaticArrays.jl#760]
```
julia> l1 = SizedArray{Tuple{2}}([1,2])
2-element SizedArray{Tuple{2},Int64,1,1} with indices SOneTo(2):
 1
 2

julia> l2 = SizedArray{Tuple{2}}([1,2])
2-element SizedArray{Tuple{2},Int64,1,1} with indices SOneTo(2):
 1
 2

julia> l1 + l2
2-element SArray{Tuple{2},Int64,1,2} with indices SOneTo(2):
 2
 4
```

**Comment by mateuszbaran** [comment]
This is a duplicate of #755.

**Comment by pjentsch0** [comment]
`st StaticArrays
Status `C:\Users\peter\.julia\environments\v1.4\Project.toml`
  [90137ffa] StaticArrays v0.12.1`

Woops. That link says it was fixed but I still get the error, I seem to be on the latest version of StaticArrays.

**Comment by mateuszbaran** [comment]
It'll be fixed in the next version. For now you can use the `master` branch.

**Comment by c42f** [comment]
Yep this will be fixed soon, I'm trying to get a new version out; #700 was blocking releases but I think I've figured out a way to unblock us there.
